package.json

```diff
@@ -60,8 +60,7 @@
     "node-sass": "^4.5.3",
     "node-static": "^0.7.9",
     "nodemon": "^1.11.0",
-    "rollup": "^0.55.0",
```

Comment:
I think `rollup` is necessary to build the bundled version (c3.js).

Reply:
It must be global because in the scripts where it's called you are using the global rollup command not from node modules, anyways i will add it back